### PR TITLE
Capacitor SDK

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #253

### CODE REVIEW

**Review by tung2744 — COMMENTED** [review]
Is it possible to start the ionic app in development / watch mode?

https://capacitorjs.com/docs/guides/live-reload

**Comment by tung2744** [comment]
It seems when `public_origin` doesn't match the entered authgear endpoint, `authenticate` will fail at the end with TypeError, but maybe this is not related to the sdk. We can open another issue to investigate it.

**Review by tung2744 — APPROVED** [review]
I think we might want to merge first and address comments later, as they are not related to the sdk directly.

File: packages/authgear-capacitor/android/src/main/java/com/authgear/capacitor/Authgear.java
[the remark this answers is not in the document]
Comment (louischan-oursky) [reply packages/authgear-capacitor/android/src/main/java/com/authgear/capacitor/Authgear.java:1]: I am not sure. I reference the official plugins https://github.com/ionic-team/capacitor-plugins/tree/main when I wrote this. Most of them are written in Java so I followed.

**Comment by louischan-oursky** [comment]
Added one more commit to enable live reload. See the commit message for details.

**Pull request merged**